go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/alexkohler/nargs
+
+go 1.21.1
```

Comment:
Best to remove the last part (.1) I think.

Reply:
done!